aviary/modules/binning/binning.smk

```diff
@@ -47,14 +48,31 @@ import os
 import sys
 import glob
 
+def get_num_samples():
```

Comment:
Is this code dupe from my recent semibin pr?

Reply:
Yep, I moved it up here